### PR TITLE
Makes malfunction AI its own dynamic ruleset instead of just traitor.

### PR DESCRIPTION
**Pull request by Putnam3145**
## About The Pull Request

Malfunctioning AI was put on the same level of threat as every other traitor, leading to silly things like nuke ops/malf at roundstart on threat level 50. This makes malf its own ruleset with its own threat level.

## Why It's Good For The Game

50 threat shouldn't be sufficient for maximum overchaos given that it's literally the *average*

## Changelog
:cl:
tweak: Malf AI is its own roundstart ruleset in dynamic mode
/:cl:

### CODE REVIEW

**Comment by Putnam3145** [comment]
draft because I haven't opened the game and made sure it works directly

File: code/game/gamemodes/dynamic/dynamic_rulesets_roundstart.dm [hunk 1]
```diff
@@ -41,6 +41,61 @@
...
+//                MALF  AI                  //
+//                                          //
+//////////////////////////////////////////////
+
+/datum/dynamic_ruleset/roundstart/delayed/malf_ai
+	name = "Malfunctioning AI"
+	config_tag = "malf_ai"
+	antag_datum = /datum/antagonist/traitor
+	antag_flag = ROLE_MALF
+	enemy_roles = list("Security Officer", "Warden","Detective","Head of Security", "Captain", "Scientist", "Chemist", "Research Director", "Chief Engineer")
```
Comment (Trilbyspaceclone) [inline code/game/gamemodes/dynamic/dynamic_rulesets_roundstart.dm:55]: chemist?

Comment (Putnam3145) [reply code/game/gamemodes/dynamic/dynamic_rulesets_roundstart.dm:55]: don't ask me, it's there by default

**Comment by Putnam3145** [comment]
Closed because subsumed into #9557

**Pull request closed without merging**